README.md

```diff
@@ -10,4 +10,43 @@ TODO: Add detailed app description and demo
 ## Architecture Diagram
 ![architecture diagram](images/architecture-diagram.png)
 [Image Link](https://excalidraw.com/#json=1c_DrTFZCSGprCvJNBYHn,B0sXrISCY8YdKgiTqy9xDA)
-_revised 9/4/2024_
```

Comment:
Request: Can we please keep this line? This will help us to keep track when changes to the architecture diagram were made.

Reply:
Yes, thought it was for the entire README doc and changed/added it to the bottom of the file instead, didn't realise it was for the architecture diagram. 

Have added back in latest commit under architecture diagram